### PR TITLE
✨ disable license check

### PR DESCRIPTION
**Pull request by laurentsimon**
* **Please check if the PR fulfills these requirements**
- [] Tests for the changes have been added (for bug fixes / features)
- [] PR title follows the guidelines defined in  https://github.com/ossf/scorecard/blob/main/CONTRIBUTING.md#pr-process

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
bug

* **What is the current behavior?** (You can also link to an open issue here)
we cannot submit any PRs

* **What is the new behavior (if this is a feature change)?**
disable the check for license header temporarily

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
no

* **Other information**:

### CODE REVIEW

**Review by inferno-chromium — APPROVED** [review]

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -45,9 +45,12 @@ jobs:
         with:
           go-version: '1.16'
       - name: Check license headers
+        # We need https://github.com/google/addlicense/pull/70#issuecomment-888099920 landed
+        # to disable testdata folers in the repo.
```
Comment (inferno-chromium) [inline .github/workflows/main.yml:49]: s/disable testdata folers/exclude testdata folders

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -45,9 +45,12 @@ jobs:
...
         run: |
-          go env -w GOFLAGS=-mod=mod
-          make all
-          set -e
-          addlicense -l apache -c 'Security Scorecard Authors' -v *
-          git diff --exit-code
+          echo "license check disabled. Re-enable as soon as possible."
```
Comment (inferno-chromium) [inline .github/workflows/main.yml:51]: s/Re-enable as soon as possible./Will be reenabled soon

File: .github/workflows/main.yml (hunk 1 above)
Comment (laurentsimon) [reply .github/workflows/main.yml:49]: ack

File: .github/workflows/main.yml (hunk 2 above)
Comment (laurentsimon) [reply .github/workflows/main.yml]: ack

**Review by azeemshaikh38 — APPROVED** [review]
Thanks Laurent!

**Pull request merged**